examples/simple-game-server/gameserver-passthrough.yaml

```diff
@@ -0,0 +1,44 @@
+# Copyright 2020 Google LLC All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: "agones.dev/v1"
+kind: GameServer
+metadata:
+  generateName: "simple-game-server-"
+spec:
+  ports:
+  - name: default
+    portPolicy: Passthrough
+  template:
+    spec:
+      containers:
+      - name: simple-game-server
+        image: gcr.io/agones-images/simple-game-server:0.1
+        env:
+        # Disables the UDP listener (Enabled by default)
+        - name: "UDP"
+          value: "FALSE"
```

Comment:
Same thing here - wondering if it's simpler to remove the env vars -- here we are demo'ing passthrough, so I don't know if we want to make things more complicated with more env varas for UDP/TCP enablement?